pkg/client/unversioned/restclient.go

```diff
@@ -19,13 +19,24 @@ package unversioned
 import (
 	"net/http"
 	"net/url"
+	"os"
+	"strconv"
 	"strings"
+	"time"
 
+	"github.com/golang/glog"
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/runtime"
 	"k8s.io/kubernetes/pkg/util"
 )
 
+const (
+	// Environment variables: Note that the duration should be long enough that the backoff
+	// persists for some reasonable time (i.e. 120 seconds).  The typical base might be "1".
+	BACKOFF_BASE     = "KUBE_CLIENT_BACKOFF_SECONDS"
```

Comment:
I was thinking this would just be a flag for now, not actual configuration. But this is fine.

Also, do not use underscores. Use mixedCaps. https://golang.org/doc/effective_go.html#mixed-caps
And there is no need for these to have public visibility.